articles/quickstart/backend/aspnet-core-webapi/01-authorization.md

```diff
@@ -13,10 +12,11 @@ github:
 contentType: tutorial
 useCase: quickstart
 ---
+<!-- markdownlint-disable MD041 MD002 -->
 
 <%= include('../../../_includes/_api_auth_intro') %>
 
-<%= include('../_includes/_api_create_new', { sampleLink: 'https://github.com/auth0-samples/auth0-aspnetcore-webapi-samples/tree/master/Samples/hs256' }) %>
+<%= include('../_includes/_api_create_new') %>
```

Comment:
I think we should also drop this SampleLink for the 2.1 sample. As well as drop the sample.

Reply:
Samples were dropped in this commit: https://github.com/auth0-samples/auth0-aspnetcore-webapi-samples/commit/4e9b146e7693213b5452fbd2c9c13e144c67eca9